.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Don't push this. We plan to integrate such things to `omniport-frontend` for consistency across the entire ecosystem.

Reply:
ok